packages/experience-builder-sdk/src/hooks/useFetchBySlug.ts

```diff
@@ -9,19 +9,23 @@ export type UseFetchBySlugArgs = {
   slug: string;
   experienceTypeId: string;
   localeCode: string;
+  hyperlinkPattern?: string;
 };
 
 export const useFetchBySlug = ({
   slug,
   localeCode,
   client,
   experienceTypeId,
+  hyperlinkPattern,
```

Comment:
I think it would make sense, but then we should put slug, localeCode etc also into it, which would need a breaking change. So I would leave it as it is for now, considering that it is already inside an object